docker-app/qfieldcloud/core/models.py

```diff
@@ -1035,6 +1039,17 @@ class Meta:
         validators=[MinValueValidator(1), MaxValueValidator(100)],
     )
 
+    # Packaging offliner to be used by the QGIS worker container.
+    packaging_offliner = models.CharField(
+        _("Packaging Offliner"),
+        help_text=_(
+            'The packaging offliner is used withing jobs to convert QGIS projects into QField projects. The new "Optimized Offliner" should be selected over the legacy "QGIS Core Offline Editing" for new projects.'
+        ),
```

Comment:
This help text should ideally match the needs for both support and endusers.

Reply:
What do you think about "[...] is used to package data for offline use with QField."?

Reply:
```suggestion
        help_text=_(
            'The packaging offliner is is used to package data for offline use with QField. The new "Optimized Packager" should be preferred over the deprecated "QGIS Core Offline Editing" for newer projects.'
        ),
```